src/index.html.js

```diff
@@ -24,9 +24,13 @@ export default function renderDocument({ helmet, rootHtml, buildManifest }) {
                 ${assets.secondary.map((asset) => `<link rel="prefetch" href="${asset.url}">`).join('\n')}
             </head>
             <body>
-                <!-- Root element where app goes -->
-                <div id="root">${rootHtml}</div>
+                <div id="root">`;
+}
+
+export function renderRemainingBlock(buildManifest) {
```

Comment:
Could we keep the same signature? (object)

Reply:
yes!